duckduckgo/iOS
# [DuckPlayer] 10 . Move DuckPlayer init to TabManager

Task/Issue URL: https://app.asana.com/0/414235014887631/1207878727316913/f

**Description**:
- Moves DuckPlayer initialization to Tabmanager so we keep only one alive instance and make DuckPlayerNavigationHandler optional for safety.



**Steps to test this PR**:

⚠️ This PR requires a specific branch of CSS to work. Use pr-releases/pr-993
⚠️ Load a custom privacy config to enable DuckPlayer from: https://jsonblob.com/api/1253721903368888320

1. CI should be 🥬
2. Smoke test DuckPlayer



**Definition of Done (Internal Only)**:
